src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 50px;
+
+}
+
+.nav__link {
+  display: flex;
+  height: 60px;
+  position: relative;
+  text-decoration: none;
+  color: black;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: calc(100% - 12px) 0;
+  margin: 20;
```

Comment:
definitely, you don't need `margin 20px`  (just 20 is invalid) around the link, maybe on one side only?